File: AR-VR/saurabhcr007_saurabhchaudhary_1923csi1081_3/GettingStarted/PrintAllPrimesTillN.java
import java.util.*;

public class PrintAllPrimesTillN {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int low=scn.nextInt();
        int high=scn.nextInt();
        int count;
        for(int i=low;i<=high;i++){
            count=0;
            for(int j=2;j*j<i;j++){
                if(i%j==0){
                    count++;
                }
                if(count>0){
                    break;
                }
            }
            if(count==0){
                System.out.println(i);
            }
        }
    }
}

File: Web/sakshiv278_SaksheeVarshney_1923co1045_3/GettingStarted/PrintAllPrimesTillN.cpp
#include <iostream>
using namespace std;
int main(int argc, char **argv){
    int low, high;
    cin >> low >> high;

    for(int n=low;n<=high;n++){
        int count=0;
        for(int i=2;i*i<=n;i++){
            if(n%i==0){
                count++;
                break;
            }
        }
        if(count==0){
            cout<<n<<endl;
        }
    }

}